Api/Controllers/AuthController.cs

```diff
@@ -27,4 +31,41 @@ public async Task<ActionResult> RegisterRestaurantOwner(RegisterRestaurantOwnerR
 
         return Ok();
     }
+
+    /// <summary>
+    /// Login authorization
+    /// </summary>
+    [HttpPost("login")]
+    [ProducesResponseType(200), ProducesResponseType(400), ProducesResponseType(401)]
+    public async Task<ActionResult> LoginUser(LoginRequest request)
+    {
+        var email = request.Email;
+        var password = request.Password;
+        
+        var user = await userManager.FindByEmailAsync(email);
+        if (user == null) return Unauthorized("Incorrect login or password.");
+        
+        var result = await signInManager.PasswordSignInAsync(email, password, false, false);
```

Comment:
Warto by było przyjmować coś w stylu bool RememberMe w LoginRequest i podawać to zamiast pierwszego false